connector/mock/connectortest.go

```diff
@@ -112,3 +112,7 @@ func (p passwordConnector) Login(ctx context.Context, s connector.Scopes, userna
 }
 
 func (p passwordConnector) Prompt() string { return "" }
+
+func (p passwordConnector) Refresh(ctx context.Context, s connector.Scopes, identity connector.Identity) (connector.Identity, error) {
```

Comment:
[nit] if `ctx` and `s` aren't used, let's go with
```go
func (p passwordConnector) Refresh(_ context.Context, _ connector.Scopes, identity connector.Identity) (connector.Identity, error) {
```

Reply:
@srenatus updated